dts/riscv/riscv32-fe310.dtsi

```diff
@@ -1,29 +1,36 @@
 /* SPDX-License-Identifier: Apache-2.0 */
 
 #include <dt-bindings/gpio/gpio.h>
+#include <freq.h>
 
 / {
 	#address-cells = <1>;
 	#size-cells = <1>;
 	compatible = "sifive,FE310G-0002-Z0-dev", "fe310-dev", "sifive-dev";
 	model = "SiFive,FE310G-0002-Z0";
+	clocks {
+		coreclk: core-clk {
+			#clock-cells = <0>;
+			compatible = "fixed-clock";
+			clock-frequency = <DT_FREQ_M(16)>;
+		};
+
+		tlclk: tl-clk {
+			#clock-cells = <0>;
+			compatible = "fixed-factor-clock";
+			clocks = <&coreclk>;
+			clock-div = <1>;
+		};
+	};
 	cpus {
 		#address-cells = <1>;
 		#size-cells = <0>;
 		cpu: cpu@0 {
-			clock-frequency = <0>;
-			compatible = "sifive,rocket0", "riscv";
 			device_type = "cpu";
-			i-cache-block-size = <64>;
-			i-cache-sets = <128>;
-			i-cache-size = <16384>;
```

Comment:
Removal of these properties  (and some below) also seems unrelated to the commit message. Can we split it out and add a proper explanation for the removal?